Source/Diagnostics/ReducedDiags/MultiReducedDiags.cpp

```diff
@@ -35,75 +35,34 @@ MultiReducedDiags::MultiReducedDiags ()
     // if names are not given, reduced diags will not be done
     if ( m_plot_rd == 0 ) { return; }
 
-    // resize
-    m_multi_rd.resize(m_rd_names.size());
-
+    using CS = const std::string& ;
+    const auto reduced_diags_dictionary =
+        std::map<std::string, std::function<std::unique_ptr<ReducedDiags>(CS)>>{
```

Comment:
Nice! Just needs a few new includes :-)
- `<map>`
- `<algorithm>`
- `<string>`
- `<functional>`
- `<memory>`
- `<iterator>`

Reply:
Yeah, that's right ! `string` and `memory` are already in `MultiReducedDiags.H`, but the others are missing...

Reply:
I should have just added them!